json_to_df.py

```diff
@@ -0,0 +1,134 @@
+
+import argparse
+import json
+import os
+
+
+import pandas as pd
+
+
+from unidecode import unidecode
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+# DIR = 'W:/DARPA_UGB/CONSULTING/MR/2018/Data/BLM/PROCESSED/PARSED/2nd_iteration'
```

Comment:
I'm going to fork this branch and remove these two lines and push another commit to take care of this. These need to be removed from the code.